tests/Browser/Operations/AssertQueryStringMissingTest.php

```diff
@@ -2,14 +2,18 @@
 
 declare(strict_types=1);
 
-test('assert query string does not have the given query param with value', function () {
-    $this->visit('/?q=test')
-        ->assertQueryStringHas('q')
-        ->assertQueryStringHas('q', 'test')
-        ->assertQueryStringMissing('q', 'test-1');
-});
+use PHPUnit\Framework\ExpectationFailedException;
+
+describe('assertQueryStringMissing', function () {
+    it('passes when the query string does not have the given query param', function () {
+        $this->visit(playgroundUrl('/?q=test'))
```

Comment:
Maybe this could be simpler.
```suggestion
        $this->visit(playground('/?q=test'))
```
Unrelated to this PR anyway, ignore for now.